temp/automatic_ticket_revocations_proposal.md

```diff
@@ -0,0 +1,123 @@
+# Proposal Structure
+
+The structure of this proposal follows the pattern established by previously proposed consensus changes.  There are two main sections of this proposal:
+
+- General Audience
+- Technical Details
+
+The general audience section provides a high-level overview of the proposal.  It includes details suitable for all stakeholders, including those unfamiliar with the inner workings of consensus.
+
+The technical details section contains a more in-depth discussion of the relevant technical aspects, implementation specifics, and rationale for the chosen semantics of the proposal.
+
+# General Audience
+
+## High-Level Overview
+
+This proposal intends to significantly improve the stakeholder user experience around ticket revocations by automatically revoking missed or expired tickets.
+
+Further, it will enable the recovery of funds for users who lost their redeem script for the legacy VSP system (before the release of [vspd](https://blog.decred.org/2020/06/02/A-More-Private-Way-to-Stake/), which removed the need for the redeem script).
+
+## Current State of Ticket Revocations
+
+Missed and expired tickets must be revoked to return the funds used to purchase the ticket to the user.  Missed and expired tickets are defined as follows:
+
+- **Missed tickets** are tickets that fail to cast a vote when selected.  Tickets are pseudorandomly selected to vote for each block.  If the wallet with voting rights for that ticket is not online to cast its vote when a ticket is selected to vote, the ticket is marked as missed.
```

Comment:
I would probably reword this a bit to account for the possibility that votes can be cast and still not included under some circumstances, such as taking too long to vote.  For example, a PoW miner might already be working on a solution to a block when the vote is cast and the miner finds the solution before generating a new template.  The details here can get pretty technical, and I don't think it makes sense to try and characterize them all here as it's mostly tangential to the DCP, but I do think it should be accurate, and the current text is only really partially correct.  Perhaps a tweak such as:

```suggestion
- **Missed tickets** are tickets that do not have a corresponding vote included in the block they are selected.  Tickets are pseudorandomly selected to vote for each block.  The most common cause of missed tickets occurs when a wallet with voting rights for that ticket is not online to cast its vote when a ticket is selected to vote.
```

Reply:
Good point, updated with your suggestion to account for that.